pkg/api/controller.go

```diff
@@ -3101,7 +3104,26 @@ func (c *Controller) GetBranch(w http.ResponseWriter, r *http.Request, repositor
 	writeResponse(w, r, http.StatusOK, response)
 }
 
-func (c *Controller) handleAPIErrorCallback(ctx context.Context, w http.ResponseWriter, r *http.Request, err error, cb func(w http.ResponseWriter, r *http.Request, code int, v interface{})) bool {
+func ConvertApiErrorToStatusCodeAndErrorMsg(logger logging.Logger, err error) (int, string, bool) {
```

Comment:
Great, one suggestion - name is too redundant.
Also please document
```suggestion
func ApiErrorToStatusCodeAndErrorMsg(logger logging.Logger, err error) (int, string, bool) {
```